### PR TITLE
Clean up MFA

### LINKED ISSUE #6: To many request to wyze server.
https://github.com/jfarmer08/homebridge-wyze-smart-home/issues/215

### PR DESCRIPTION
**Pull request by jfarmer08**
Clean up MFA and run pretty code from VS

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -160,44 +158,40 @@ module.exports = class WyzeAPI {
...
+      throw new Error(
+        'KeyId Required, Please provide the "keyid" parameter in config.json'
+      );
+    } else {
+      try {
+        result = await this._performLoginRequest();
+        if (this.apiLogEnabled)
+          this.log.debug("Successfully logged into Wyze API");
+        await this._updateTokens(result.data);
+      } catch (error) {
```
Comment (hgoscenski) [inline src/index.js:189]: Do we want to log the specific error in the error message here, provide a hint for the user to track down the issue?

**Review by hgoscenski — APPROVED** [review]
lgtm, we may want to log login error for end users.

File: src/index.js (hunk 1 above)
Comment (jfarmer08) [reply src/index.js:189]: That would be good.

**Comment by hgoscenski** [comment]
@jfarmer08 would you like me to merge?

**Comment by jfarmer08** [comment]
Sure can.

Thanks Allen :)

**Pull request merged**